File: User Services/freelancer/src/main/java/com/worksphere/freelancer/entity/Role.java
package com.worksphere.freelancer.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "role")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Role {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "rid")
    private Integer rid;

    @Column(name = "rname")
    private String rname;
}

File: RestAPIs/Admin_Services/Models/Role.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Admin_Services.Models
{
    public class Role
    {
        [Key]
        [Column("rid")]
        public int Rid { get; set; }

        [Column("rname")]
        public string Rname { get; set; } = null!;
    }
}
